com.incquerylabs.v4md/build.gradle

```diff
@@ -343,7 +334,8 @@ publishing {
 
     publications {
         mavenJava(MavenPublication) {
-            artifactId "${artifactId}-${mdVersion}"
+        	artifactId "${artifactId}-${mdVersion}"
+        	
```

Comment:
Why did this line change?

Reply:
may have changed while merge conflict resolving